docs/pages/reference/backends.mdx

```diff
@@ -1429,3 +1429,90 @@ the location set the `data_dir` value within the Teleport configuration file.
 teleport:
   data_dir: /var/lib/teleport_data
 ```
+
+## CockroachDB
+
+Teleport can use [CockroachDB](https://www.cockroachlabs.com/) as a storage backend
+to achieve high available and survive regional failures. You must take steps to 
+protect access to CockroachDB in this configuration because that is where Teleport
+secrets like keys and user records will be stored.
+
+<Admonition
+  type="warning"
+  title="IMPORTANT"
+>
+  CockroachDB can currently only be used to store Teleport's cluster state.
```

Comment:
Starting to feel like we should have a matrix on this page that lists each backend and each use case (cluster state, audit log, session recordings, etc) to indicate which backends support which types of data.